content/developer/reference/user_interface/views/include/form.rst

```diff
@@ -0,0 +1,778 @@
+.. _reference/user_interface/views/form:
+
+Form
+====
+
+Form views are used to display the data from a single record. Their root element is
+``<form>``. They are composed of regular HTML_ with additional structural and semantic
+components.
+
+.. code-block:: xml
+
+  <form>
+    ...
+  </form>
+
+Optional attributes_ are added on root element ``<form>`` to customize the view.
+
+:string:
+  string_
+
+  View name
+
+:create:
+  boolean_
+
+  Disable/enable record creation on the view.
+
+:edit:
+  boolean_
+
+  Disable/enable record editing on the view.
+
+:duplicate:
+  boolean_
+
+  Disable/enable record duplication on the view through the **Action** dropdown.
+
+:delete:
+  boolean_
+
+  Disable/enable record deletion on the view through the **Action** dropdown.
+
+:js_class:
+  string_
+
+  Name of the javascript class to use when instantiating the view.
+
+:disable_autofocus:
+  boolean_
+
+  Disable automatic focus of the first field in the view.
+
+.. _reference/user_interface/views/form/semantic:
+
+Semantic components
+-------------------
+
+Semantic components tie into and allow interaction with the Odoo system.
+Placeholders are denoted in all caps.
+
+.. _reference/user_interface/views/form/field:
+
+<field>: render formatted values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: xml
+
+  <form>
+    <field name="FIELD_NAME"/>
+  </form>
+
+renders (and allow editing of, possibly) a single field of the current
+record. Using several times a field in a form view is supported and the fields
+can receive different values for modifiers 'invisible' and 'readonly'. However,
```

Comment:
shouldn't fields receive different values for 'invisible' so only one is visible at a time ?